js/core/index.js

```diff
@@ -43,3 +57,44 @@ function Runtime() {
 }
 
 global.runtime = module.exports = new Runtime();
+
+runtime.tty.stdout = new runtime.tty.StdOut();
+runtime.tty.stdin = new runtime.tty.StdIn();
+initio(runtime);
+
+runtime.stdio = {
+  stdout: runtime.tty.stdout,
+  stdin: runtime.tty.stdin,
+  StdIn: runtime.tty.StdIn,
+  StdOut: runtime.tty.StdOut,
+  createOut: function(funcs) {
```

Comment:
I'm not sure if we should support convenience functions, let's just provide a raw building blocks for other libraries or streaming interfaces to build on top. It might be confusing to figure out the difference between `new StdOut` and `createOut`. What do you think?